src/modules/dashboard/components/navigation-drawer.vue

```diff
@@ -48,9 +52,15 @@ onClickOutside(navigationDrawer, (e: MouseEvent) => {
 const { openDialog, closeDialog } = useDialog();
 
 function handleLogout(): void {
-  openDialog('Logout', 'Are you sure you want to logout?', () => {
-    userStore.logout();
-    closeDialog();
-  });
+  const dialogOptions: IDialogOptions = {
+    title: i18n.global.t('Logout'),
+    message: '',
```

Comment:
Jeżeli mamy case gdzie nie chcemy mieć message, to może zrobimy go opcjonalny w obiekcie konfiguracyjnym?